google/cloud/pubsublite/cloudpubsub/internal/single_partition_subscriber.py

```diff
@@ -75,13 +90,14 @@ async def read(self) -> Message:
             )
             cps_message = self._transformer.transform(message)
             offset = message.cursor.offset
+            ack_id = f"{self._ack_generation_id}/{offset}"
```

Comment:
can you instead break this out into a simple class with parse(payload: str) and __str__ methods? You can see MessageMetadata as a template for this.